esp32-led-blink-sdk/main/CMakeLists.txt

```diff
@@ -11,6 +31,8 @@ add_custom_command(
         ${SWIFTC}
         -target riscv32-none-none-eabi
         -Xfrontend -function-sections -enable-experimental-feature Embedded -wmo -parse-as-library -Osize
+        -Xcc -march=${march_flag} -Xcc -mabi=${mabi_flag}
+        -Xlinker -march=${march_flag} -Xlinker -mabi=${mabi_flag}
```

Comment:
Do you mind explaining why the linker also needs this information?

Reply:
It doesn't need. Removed.